common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -307,7 +308,7 @@ if Backbone?
         model: @model
         mode: @mode
         context: @context
-        course_settings: @options.course_settings
+        course_settings: @course_settings
```

Comment:
This was changed due to this error: `TypeError: this.options is undefined in http://0.0.0.0:49615/suite/common/include/coffee/src/discussion/views/discussion_thread_view.js (line 1392)
` in this jasmine test `JavaScript.firefox.DiscussionThreadView inline mode expand/collapse: re-renders the show view correctly when leaving the edit view (from javascript)
`